server/app/featureflags/FeatureFlags.java

```diff
@@ -58,6 +60,15 @@ public boolean isProgramEligibilityConditionsEnabled() {
     return config.getBoolean(PROGRAM_ELIGIBILITY_CONDITIONS_ENABLED);
   }
 
+  /**
+   * If specifying multiple questions in a predicate is enabled.
+   *
+   * <p>Allows for overrides set in {@code request}.
+   */
+  public boolean isPredicatesMultipleQuestionsEnabled(Request request) {
```

Comment:
There's an easy-ish test to update for this too. 

https://sourcegraph.com/github.com/civiform/civiform/-/blob/server/test/featureflags/FeatureFlagsTest.java?subtree=true

Just add it to the first couple that check these conditions together for all flags.  

the program read only one could be folded in to it too but wasn't.

Reply:
done